scripts/generate_events.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+go run ./tools/eventgen/*.go
```

Comment:
Could you please expand on the doc comment explaining what this script does, and what its output is? It isn't immediately obvious.

Reply:
👍